### DISCUSSION TITLE
Welcome | Introductions

### DISCUSSION THREAD

**Discussion by johanjanssens** [joomlatools/joomlatools-pages#495] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us a bit about how your are using Pages.

**Comment by JoeJoomla** [comment]
<p>I'm Joe Sonne and I've been using Joomla since 2005.</p>
<p>I'm just getting started with Pages and used it for the first time on a client site on this page:<br /><a href="https://campusforcommunities.ca/resources/facilitative-tools" rel="nofollow noopener noreferrer" target="_blank">https://campusforcommunities.ca/resources/facilitative-tools</a></p>
<p>I posted a little more information about the above in the <strong>show and tell</strong> section here: <a href="https://github.com/joomlatools/joomlatools-pages/discussions/496#discussion-82128" rel="nofollow noopener noreferrer" target="_blank">Pages Page | Campus For Communities</a></p>
<p>Most of my clients already have mature Joomla websites so integrating Pages into their existing sites means you have to accommodate things for their benefit. In the above case the client needed to be able to edit their content so the content in the Page is picked up from standard Joomla articles. The speed of the Page rocks considering how graphic intense it is.</p>
<p>My initial thoughts on Pages is that it is excellent and I have plans to use it for full sites in the near future.</p>

**Reply by johanjanssens** [reply]
Welcome Joe!

**Comment by waseemsadiq** [comment]
Hi,

I'm Waseem Sadiq, I've been building sites with and using Joomla since 2005 (this feels a bit like an AA meeting @JoeJoomla 🤣).

I've been a consultant at timble, Johan's company, since 2017 and I helped to build https://www.agrea.ph - our first client website built on Pages.

I'll be helping out where I can.

**Reply by johanjanssens** [reply]
Thanks man for jumping in. Think we are more GG (glorious geeks) ;)

**Comment by kbrookes** [comment]
Hi All,

I'm Kelsey Brookes, I've been a Joomla site builder since the early 2000's, using Mambo before The Great Schism(TM). I run a brand agency with my business partner, and I've always preferred Joomla, largely due to the override system which allowed me to have greater control over the layout of the sites we build.

Loving Pages - my first client site is running a 0.18 version here: https://www.distilledconsulting.com.au/

Helping out with documentation were I can, as I'm not really at the level where my code contributions would help anyone!

K

**Reply by johanjanssens** [reply]
Thanks Kelsey, keep the feedback coming!
